.babelrc

```diff
@@ -6,6 +6,7 @@
   ],
   "presets": ["es2015", "stage-2"],
   "plugins": [
+    "jsx-control-statements",
```

Comment:
This is a cool project that lets us do stuff like `<If condition={foo}><div /></If>` without any overhead (it's converted into a ternary expression). I think it improves readability a lot.